data/apply-schema-update.sql

```diff
@@ -1,94 +1,113 @@
+CREATE OR REPLACE function tmp_add_column(tbl_name text, cln_name text, typ_name text)
```

Comment:
Can you explain the name of this function more? Seems like the column is persistent so seems like a weird name.